server/models/Professional.ts

```diff
@@ -0,0 +1,51 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import {User} from './User';
+
+export enum Race {
+    WHITE = "white",
+    BLACK = "black",
+    ASIAN = "asian",
+    LATINX = "latinx",
+    INDIGENOUS = "indigenous",
+    DECLINE = "decline",
+    GHOST ="ghost"
+}
+
+export enum Education {
+    ASSOCIATES = "associates",
+    BACHELORS = "bachelors",
+    MASTERS = "masters",
+    PHD = "doctorate",
+    CERT = "industry certification",
+    GHOST = "ghost"
+}
+
+
+@Entity()
+export class Professional extends User {
+  @PrimaryGeneratedColumn()
+  id!: number;
+
+  @Column({ length: 100, nullable: false })
+  name!: string;
+
+  @Column({ nullable: false })
+  password!: string;
+
+  @Column({ nullable: false, length: 512 })
+  email!: string;
+
+  @Column({
+    type: 'enum', 
+    enum: Race,
+    default: Race.GHOST 
```

Comment:
what is the role of the Ghost enum here? Is it to represent the absence of a value?  If so, we can just make these fields nullable, rather than having a new enum for it